### ISSUE TITLE
Request for ComplEx Embedding

### ISSUE THREAD

**Issue by weikai-li** [ICT-GIMLab/SeHGNN#3]
Hello!
I generated the ComplEx embedding as you instructed and ran your code. I basically reproduced your results of "SeHGNN" but could not reproduce the results of "ComplEx + SeHGNN." My first-stage result is about 55.83(valid set) and 54.12(test set), much lower than yours. I wonder whether it has something to do with the ComplEx embedding. Could you please provide your generated ComplEx embedding? Thank you!

**Comment by Yangxc13** [comment]
Thanks for your attention and help. Please change the '--max_step 10' (number of maximum training steps) to a large number such as 500000 in the file `./data/complex_nars/train_graph_emb.sh` for embedding generation. This bug has been fixed in the lastest commit.

---

About why this bug occurred: Before releasing the codes, we decreased this number for quickly verifying the smoothness of the whole training process, but sorry we forgot to change it back.

**Comment by weikai-li** [comment]
Thank you! I have successfully reproduced your results.

**Comment by weikai-li** [comment]
Thank you! I have successfully reproduced your results.

在2022-08-09 ***@***.***写道：

Thanks for your attention and help. Please change the '--max_step 10' (number of maximum training steps) to a large number such as 500000 in the file ./data/complex_nars/train_graph_emb.sh for embedding generation. This bug has been fixed in the lastest commit.

About why this bug occurred: Before releasing the codes, we decreased this number for quickly verifying the smoothness of the whole training process, but sorry we forgot to change it back.
